src/components/App.js

```diff
@@ -41,11 +42,14 @@ class App extends Component {
         <TopBar />
         <Grid container>
           <Grid item md={9} sm={12} xs={12} className={classes.calendar}>
-            <CalendarView />
+            <ClassesView />
           </Grid>
           <Grid item md={3} sm={12} xs={12}>
             <SidebarView />
           </Grid>
+          <Grid item md={9} sm={12} xs={12}>
+            <UnscheduledClasses />
```

Comment:
I think you forgot to take this out after putting in the tab.

Reply:
oops that was sloppy - fixed